docker/ubuntu/app/setup.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# Install softwares
+apt-get update -y
+apt-get -y -o APT::Immediate-Configure=false install wget
+wget -O- http://archive.apache.org/dist/bigtop/bigtop-1.1.0/repos/GPG-KEY-bigtop | sudo apt-key add -
+wget -O /etc/apt/sources.list.d/bigtop-1.1.0.list http://archive.apache.org/dist/bigtop/bigtop-1.1.0/repos/`lsb_release --codename --short`/bigtop.list
+
+# for openjdk-8-jre-headless
+apt-get -y install software-properties-common
+add-apt-repository ppa:openjdk-r/ppa
+apt-get update -y
+
+apt-get install -y -q --no-install-recommends openjdk-8-jre-headless vim screen curl sudo unzip man openssh-server hadoop\*
+
+wget https://ci.bigtop.apache.org/job/Bigtop-trunk-packages/COMPONENTS=apex,OS=ubuntu-16.04/lastSuccessfulBuild/artifact/output/apex/apex_3.6.0-1_all.deb
```

Comment:
Updated this PR with the atrato release. I can create a docker image for 3.7.0 using apex-core/master and only then I'll go about working on APEXCORE-813 (using binary from apex-core instead of atrato). Hope that is fine.

Reply:
The `3.7.0` version should still be created from your repo, so that we have a release branch (vs. master). We can then switch to dockerhub to build from the Apache report for `latest`.

The only thing that needs to be done prior to creating the `3.7.0` image is to change the apex package portion as done in this PR (rest was already tested).